exercises/luhn/luhn.el

```diff
@@ -0,0 +1,11 @@
+;;; luhn.el --- Luhn exercise (exercism)
+
+;;; Commentary:
+
+;;; Code:
+
```

Comment:
Just for consistencies sake with the rest of the codebase, can we keep this to two blank lines between the `;; Code:` block and the `provide` statement?